src/jenkins_job_insight/main.py

```
@@ -527,13 +540,45 @@ async def lifespan(app: FastAPI):
+                    task = asyncio.create_task(
+                        dispatch_alert(
+                            "high_error_rate",
+                            f"\u26a0\ufe0f JJI high error rate: {snap['error_rate']:.0%} "
+                            f"({snap['total_errors']}/{snap['total_requests']} requests "
+                            f"in {snap['window_seconds']}s window)",
+                        )
+                    )
+                    _background_tasks.add(task)
+                    task.add_done_callback(_background_tasks.discard)
+            except Exception:
+                pass  # Never crash the app for alerting
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)